app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,25 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = Recipe.all.where(user: current_user)
+  end
+
+  def show
+    @recipe = Recipe.find(params[:id])
+    @recipes_food = RecipeFood.all.where(recipe: @recipe)
+  end
+
+  def destroy
+    @recipe = Recipe.find(params[:id]).destroy!
+    redirect_to recipes_path, notice: 'Recipe has been deleted successfully!'
+  end
+
+  def toggle
+    @recipe = Recipe.find(params[:id])
+    @recipe.toggle!(:public)
+    redirect_to recipes_detail_path(params[:id])
+  end
+
+  def public_recipes
+    @recipes = Recipe.all.where(public: true)
+  end
+end
```

Comment:
- [ ] As requested by the previous reviewer, please make sure there is a way to create recipes, I think you are missing the create recipe method in the RecipesController, and also missing the form to create recipe in the views, so just like you have it for the FoodsController, please make sure the user is able to create recipes :+1: 

currently we get an empty view with no action that a user can take on the `/recipes` path please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230737824-80449de6-f209-419a-9bdc-2c7dde89b64e.png)

according to the wireframe /recipes path should display the recipes belonging to the user along with a button or link to create a recipe which will take us to the form to create a recipe, and then clicking on each recipe will take us to that particular recipes detail page showing the details of the recipe and an option to generate shopping list like this :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230737985-2c0a7d6f-74a6-49a2-b29e-8a39eebf9a51.png)